### PR TITLE
first upload of calcQuality function

### PR DESCRIPTION
**Pull request by clairezurn**
Uploading function calcQuality for the src folder.

### CODE REVIEW

File: pyautomagic/src/calcQuality.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    OUTPUTS: quality_metrics, a dictionary of the quality metrics and thresholds/settings used
+    The four quality metrics:
+        Overall high amplitude data points
+        Timepoints of high variance across channels
+        Ratio of bad channels
+        Channels of high variance across time
+        
+        Mean absolute voltage also calculated
+    """
+    import numpy as np
```
Comment (adam2392) [inline pyautomagic/src/calcQuality.py]: Put import statements at top of the file

**Comment by adam2392** [comment]
Also, if you can, write a short unit test to see how that works. 

Stick it in the tests/pyautomagic/ folder. You can just simulate dummy data that you should know the answer. Should be pretty simple, since you're doing a simple computation.

**Comment by clairezurn** [comment]
Updates suggested were made, test file added.

**Comment by adam2392** [comment]
@saulmezac try reviewing this code and giving either constructive criticism or saying why it should be approved.

File: pyautomagic/src/calcQuality.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        timeThresh = 25
+        logger.log(30,'Invalid timeThresh value. Default of 25 used.')
+    if not isinstance(chanThresh, int) and not isinstance(chanThresh, float) or isinstance(chanThresh,bool) or (chanThresh <= 0):
+        chanThresh = 25
+        logger.log(30,'Invalid chanThresh value. Default of 25 used.')
+    if not isinstance(apply_common_avg, bool):
+        # ADD STATEMENT TO ALLOW TYPICAL OTHER BOOLEAN INDICATORS
+        apply_common_avg = True
+        logger.log(30,'Invalid apply_common_avg value. Average referencing used.')
+    # ADD CHECKS FOR DAT AND BAD_CHANS?
```
Comment (saulmezac) [inline pyautomagic/src/calcQuality.py]: Will you do the verification for dat and bad_chans? If not, you can erase this comment

**Comment by saulmezac** [comment]
Seems fine to me, overall code should work fine, you made all the changes requested by Adam before and some other changes. If your Unit Test shows that the code is working, then this is ready to merge with master.

**Pull request merged**